scripts/cibuild.sh

```diff
@@ -53,11 +53,27 @@ if [[ ! -z $SYNTAX ]]; then
 	exit 1
 fi
 
+echo -e "\n>> Starting Selenium in background..."
+invoke start --service selenium
+RC=$?
+if [[ $RC != 0 ]]; then
+	echo -e "\n$(info) Selenium failed to start."
+	exit $RC
+fi
+
 echo -e "\n>> Running unit tests..."
-invoke unittest --failfast
+invoke unittest --failfast --keepdb
+RC=$?
+if [[ $RC != 0 ]]; then
+	echo -e "\n$(info) one or more unit tests failed, failing build."
+	exit $RC
+fi
+
+echo -e "\n>> Running integration tests..."
+invoke integration-test --failfast --keepdb
```

Comment:
Looks like CI is now reporting only the test coverage from the integration test alone. I think we need to add a `--append` option to the invoke task, that would be passed through as `coverage run --append`, so that we get the consolidated coverage numbers.

Reply:
Huh okay, good catch!